modules/stanford_events/lib/scss/stanford_events.node.scss

```diff
@@ -86,10 +86,17 @@
     }
 
     .su-event-type-bottom {
+      &.su-event-type {
+        div {
+          padding: 0.1em 0.3em 0.1em 0;
```

Comment:
This needs to be on the link, not the div. the div has no affect on touch target. A touch target is a button, link, input, etc.

Reply:
Ok. I removed it from the `div` and added to the `a`. I added it to the top term as well and paid attention to the commas not moving if multiples at the top.